source/documentation/tools/quicksight/04-working-with-quicksight.md

```diff
@@ -0,0 +1,44 @@
+# Working with QuickSight
+
+## Finding datasets
+Datasets are available via the `Athena` data source in QuickSight: 
+
+>Datasets > New Dataset (top right) > Athena > New Athena data source.
+
+When adding a new dataset via Athena you'll be prompted for a `Data source name`. 
+This is just a descriptive name for the new data source tile for **you** in **your** QuickSight interface.
+
+Here's [the advice from aws] about naming your Data source:
+
+> _This name displays on the Amazon QuickSight list of existing data sources, which is at the bottom of the 'Create a Dataset' view. 
+Use a name that makes it easy to distinguish your data sources from other similar data sources._
+
+Unless specified elsewhere for your data, use the default `Athena workgroup`.
+
+Data will be in the default Catalog (`AwsDataCatalog`), and you should only see Databases that your user has access to via the Database Access Repo.
+
+Adding a dataset in this way enables you to add a single table at a time.
+
+You'll be asked if you want to 'Import to [SPICE] for quicker analytics'.  
+This imports the data into memory, rather than querying the data where it lives in S3. Whether to use SPICE or directly query the data is a judgement call, and will likely come down to:
+
+- how complex the analysis is
+- the size, format and structure of the underlying data
+- how likely users are to drill down into the dashboards
+- user tolerance for load times while making changes to dashboards
```

Comment:
Is it possible to flip this approach? For example, giving users a recommendation for the factors you've described. 'You should not use SPICE if...'

Reply:
I like this idea. My suggestion would basically boil down to 'SPICE costs money, don't use it if you don't need it'. Due to lack of familiarity with the impacts it makes on user experience, I don't feel knowledgable enough for that approach, and would prefer to leave it in the users' hands.